scripts/find_licensee.rb

```diff
@@ -0,0 +1,5 @@
+licensees = Licensee.where ('eye_color = :eye_color' AND 'hair_color = :hair_color'),
+                  eye_color: 'Hazel'
+                  hair_color: 'Brown', 'Black'
```

Comment:
Not quite. You could have written
`Licensee.where eye_color: 'Hazel', hair_color: ['Brown', 'Black']`
But because you used a query string, you can't use a set of values - it won't translate correctly. You'd need to specify the logic of the query explicitly:
`Licensee.where 'eye_color = :ec AND (hair_color = :hc1 OR hair_color = :hc2)', ec: 'Hazel', hc1: 'Brown', hc2: 'Black'`